packages/edit-site/src/components/block-editor/block-editor-provider/default-block-editor-provider.js

```diff
@@ -1,75 +0,0 @@
-/**
```

Comment:
The logic that was here need to be absorbed by the `EditorProvider` from the `@wordpress/editor` package so I'm trying to add a `templateMode` prop there to support that. I added a prop but I'm not there yet, I first need to understand the logic and what are all the modes, how to trigger them...